### PR TITLE
✨ Add ApplePressAndHoldEnabled command

### PR DESCRIPTION
**Pull request by Gudsfile**
Add command `ApplePressAndHoldEnabled`

> Choose the behavior of a key held down

- Tested on Monterey and Big Sur only
- Added in english and french languages
- ⚠️ Scripts to build screenshots **not provided** (trouble with the node-pngquant-native module and Mac with Apple silicon)
- Manual screenshot added

### CODE REVIEW

File: defaults.yml [hunk 1]
```diff
@@ -1027,3 +1027,36 @@ categories:
           - value: false
             text: '"Application Downloaded from Internet" popup will not display'
         versions: [Catalina]
+      - key: ApplePressAndHoldEnabled
+        domain: NSGlobalDomain
+        title: Choose the behavior of a key held down
```
Comment (yannbertrand) [inline defaults.yml:1032]: Shorter title suggestion
```suggestion
        title: Key held down behavior
```

File: defaults-fr.yml [hunk 2]
```diff
@@ -707,3 +707,30 @@ categories:
             text: L'affichage des notifications est activé.
         versions: [Catalina]
         after: killall Music
+      - key: ApplePressAndHoldEnabled
+        domain: NSGlobalDomain
+        title: Choisir le comportement lorsqu'une touche est maintenue enfoncée
```
Comment (yannbertrand) [inline defaults-fr.yml:712]:
```suggestion
        title: Comportement touche maintenue enfoncée
```

File: defaults-fr.yml [hunk 3]
```diff
@@ -707,3 +707,30 @@ categories:
...
+              width: 696
+              height: 252
+            text: Par défaut, lorsqu'une touche est maintenue enfoncée, le menu des accents s'affiche.
+          - value: false
+            image:
+              filename: false.png
+              width: 696
+              height: 252
+            text: Répète la touche tant que celle-ci est maintenue enfoncée.
+        versions: [Monterey, Big Sur]
```
Comment (yannbertrand) [inline defaults-fr.yml:736]: Not sure why GitHub doesn't show it but you added blank lines at the end of the file 😬 that's what's causing the failed build.

Please remove them :)

Comment (Gudsfile) [reply defaults-fr.yml:736]: Oops 🫣 It's done

**Comment by yannbertrand** [comment]
Thanks a lot 😄

**Pull request merged**